src/test/java/posters/flows/OpenLoginPageFlow.java

```diff
@@ -10,12 +9,12 @@
     public static LoginPage flow()
     {
         // initialize the session and go to home page
-        HomePage homePage = OpenHomePageFlow.flow();
+        var homePage = OpenHomePageFlow.flow();
 
         // open login page and check for expected page
-        LoginPage loginPage = homePage.userMenu.openLogin();
+        var loginPage = homePage.userMenu.openLogin();
         loginPage.isExpectedPage();
 
         return loginPage;
```

Comment:
can be shortened from three lines to one:
```
return homePage.userMenu.openLogin().isExpectedPage();
```
at least this variant should be used if you want to keep the readability high:
```
var loginPage = homePage.userMenu.openLogin();
return loginPage.isExpectedPage();
```

Reply:
@ufjena This hs not been covered yet